src/snapred/backend/data/LocalDataService.py

```diff
@@ -849,159 +517,40 @@ def readReductionData(self, runNumber: str, useLiteMode: bool, version: int) ->
         logger.info(f"loaded reduction data from {dataFilePath}: version: {version}")
         return record
 
-    ##### CALIBRANT SAMPLE METHODS #####
-
-    def readSampleFilePaths(self):
-        sampleFolder = Config["instrument.calibration.sample.home"]
-        extensions = Config["instrument.calibration.sample.extensions"]
-        # collect list of all json in folder
-        sampleFiles = set()
-        for extension in extensions:
-            sampleFiles.update(self._findMatchingFileList(f"{sampleFolder}/*.{extension}", throws=False))
-        if len(sampleFiles) < 1:
-            raise RuntimeError(f"No samples found in {sampleFolder} for extensions {extensions}")
-        sampleFiles = list(sampleFiles)
-        sampleFiles.sort()
-        return sampleFiles
-
-    def writeCalibrantSample(self, sample: CalibrantSamples):
-        samplePath: str = Config["samples.home"]
-        fileName: str = sample.name + "_" + sample.unique_id
-        filePath = os.path.join(samplePath, fileName) + ".json"
-        if os.path.exists(filePath):
-            raise ValueError(f"the file '{filePath}' already exists")
-        write_model_pretty(sample, filePath)
-
-    def readCalibrantSample(self, filePath: str):
-        if not os.path.exists(filePath):
-            raise ValueError(f"The file '{filePath}' does not exist")
-        with open(filePath, "r") as file:
-            sampleJson = json.load(file)
-            if "mass-density" in sampleJson and "packingFraction" in sampleJson:
-                warnings.warn(  # noqa: F821
-                    "Can't specify both mass-density and packing fraction for single-element materials"
-                )  # noqa: F821
-            del sampleJson["material"]["packingFraction"]
-            for atom in sampleJson["crystallography"]["atoms"]:
-                atom["symbol"] = atom.pop("atom_type")
-                atom["coordinates"] = atom.pop("atom_coordinates")
-                atom["siteOccupationFactor"] = atom.pop("site_occupation_factor")
-            sample = CalibrantSamples.model_validate_json(json.dumps(sampleJson))
-            return sample
-
-    def readCifFilePath(self, sampleId: str):
-        samplePath: str = Config["samples.home"]
-        fileName: str = sampleId + ".json"
-        filePath = os.path.join(samplePath, fileName)
-        if not os.path.exists(filePath):
-            raise ValueError(f"the file '{filePath}' does not exist")
-        with open(filePath, "r") as f:
-            calibrantSampleDict = json.load(f)
-        filePath = Path(calibrantSampleDict["crystallography"]["cifFile"])
-        # Allow relative paths:
-        if not filePath.is_absolute():
-            filePath = Path(Config["samples.home"]).joinpath(filePath)
-        return str(filePath)
```

Comment:
These were not edited, this section was simply moved to below the read / write state methods.